app/group.go

```diff
@@ -94,10 +103,51 @@ func (a *App) CreateGroup(group *model.Group) (*model.Group, *model.AppError) {
 	return group, nil
 }
 
+func (a *App) CreateGroupWithUserIds(group *model.GroupWithUserIds) (*model.Group, *model.AppError) {
+	newGroup, err := a.Srv().Store.Group().CreateWithUserIds(group)
+	if err != nil {
+		var invErr *store.ErrInvalidInput
+		var appErr *model.AppError
+		var dupKey *store.ErrUniqueConstraint
+		switch {
+		case errors.As(err, &appErr):
+			return nil, appErr
+		case errors.As(err, &invErr):
+			return nil, model.NewAppError("CreateGroupWithUserIds", "app.group.id.app_error", nil, invErr.Error(), http.StatusBadRequest)
+		case errors.As(err, &dupKey):
+			return nil, model.NewAppError("CreateGroup", "app.custom_group.unique_name", nil, dupKey.Error(), http.StatusBadRequest)
+		default:
+			return nil, model.NewAppError("CreateGroupWithUserIds", "app.insert_error", nil, err.Error(), http.StatusInternalServerError)
+		}
+	}
+
+	messageWs := model.NewWebSocketEvent(model.WebsocketEventReceivedGroup, "", "", "", nil)
```

Comment:
Who should this message reach to? Just the affected users or more?

Reply:
I think this is the same reason as https://github.com/mattermost/mattermost-server/pull/18839#discussion_r770631824. @BenCookie95?

Reply:
I actually didn't need the websocket for creating a group because we make a network request when we're typing to search for groups

Reply:
Ok, I can remove this one until it ever becomes necessary.